apps/server/src/modules/task-card/uc/task-card.uc.ts

```diff
@@ -191,7 +191,9 @@ export class TaskCardUc {
 	}
 
 	private checkSchoolYearEndDate(schoolYearEndDate: Date, dueDate: Date) {
-		if (schoolYearEndDate < dueDate) {
+		const schoolYearEndDateIgnoresTime = new Date(schoolYearEndDate).setHours(0, 0, 0, 0);
+		const dueDateWithoutIgnoresTime = new Date(dueDate).setHours(0, 0, 0, 0);
```

Comment:
`dueDateWithoutIgnoresTime` especially confuses me because the "without" and "ignoresTime" somehow seems to contradict itself